### ISSUE TITLE
Add performance tracking

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-code-manager#1423]
Performance is an issue at times in the app. In order to get a better handle on exactly how often performance concerns occur, we intend to add performance tracking which will contribute data to InfluxDB.

This issue is about investigating how to do that, and adding the code to record the metrics that concern us.

**Comment by bobsilverberg** [comment]
I realized that we can use the React [Profiler API](https://reactjs.org/docs/profiler.html) to do this (which I believe @muffinresearch suggested some time ago). I thought the React Profiler only worked for interactive profiling in the browser, but there is an API that can be used to capture rendering performance. I am working on a patch using this.

**Comment by ioanarusiczki** [comment]
@bobsilverberg Thank you for the detailed instructions. 

- I did not count the events, they seem to be plenty  so after landing on a new Browse/Compare file I selected a couple of events from the Console and matched them with it's equivalent entry from the Network panel.  I found this way a match for the `utl` with the `label` and `utt` with the `value`.

- There seem to be plenty of such beacons sent not only when a new page is loaded but while doing an action on the page - simply clicking the Compare Versions button, collapse the panel, information or making a comment.

![plenty of events](https://user-images.githubusercontent.com/33448286/78140654-02f53580-7433-11ea-91cd-95371b10a786.gif)

I believe this works as expected so I'm going to mark the issue verified. - FF74(Win10).
